lib/qrda-import/base-importers/section_importer.rb

```diff
@@ -79,6 +80,9 @@ def extract_codes(coded_element, code_xpath)
       end
 
       def code_if_present(code_element)
+        if code_element && code_element['code'].nil? && code_element['sdtc:valueSet'].nil?
```

Comment:
This check will want to be in the "extract_negated_code" method instead.  The "code_if_present" is used for all coded values (some of which would be ok to nullFlavor).

Reply:
Ahh- ok. So we only really want this for when it's implied that we're negating a valueset, but then a valueset isn't included?